docs/cloud/pouta/additional-services.md

```diff
@@ -38,12 +38,25 @@ the CSC Service Desk to coordinate this.
 
 cPouta does not currently offer integrated name service management.
 
-All our floating IPs are mapped to a default hostname, for example
+All our floating IPs are mapped to a default hostname, for example:
 
 ```
 vm0120.kaj.pouta.csc.fi
 ```
 
+To find hostname of a floating IP, you could leverage `host` command:
```

Comment:
You can add "the" in front of "hostname" -> "To find _the_ hostname of a floating IP..."

I would also rephrase "you could leverage host command" to a simpler "you can use the host command" or "you can leverage the host command"

Reply:
addressed, thanks